pallets/cash/src/internal/events.rs

```diff
@@ -259,39 +259,51 @@ pub fn formulate_reorg<T: Config>(
     chain_id: ChainId,
     last_block: &ChainBlock,
     true_block: &ChainBlock,
+    chunk_size: u32,
 ) -> Result<ChainReorg, Reason> {
     let starport = get_starport::<T>(chain_id)?;
     let mut reverse_blocks: Vec<ChainBlock> = vec![];
     let mut drawrof_blocks: Vec<ChainBlock> = vec![];
```

Comment:
Can we just add a comment here `// drawrof is "forward" backwards, since these are forward blocks stored in reverse order` or something?

Reply:
I added a comment about that its just further down